src/map/ai/states/item_state.cpp

```diff
@@ -97,7 +97,7 @@ CItemState::CItemState(CCharEntity* PEntity, uint16 targid, uint8 loc, uint8 slo
         }
         else
         {
-            throw CStateInitException(std::make_unique<CMessageBasicPacket>(m_PEntity, m_PEntity, param, 0, error == -1 ? 0 : error));
+            throw CStateInitException(std::make_unique<CMessageBasicPacket>(m_PEntity, PTarget, param, 0, error == -1 ? 0 : error));
```

Comment:
we did something similar in magic_state, but it would cause a crash because sometimes PTarget was null - can you do PTarget ? PTarget : m_PEntity instead of just PTarget?